connector/connect/common/src/main/protobuf/spark/connect/expressions.proto

```diff
@@ -37,6 +37,26 @@ message Expression {
     Alias alias = 6;
     Cast cast = 7;
     UnresolvedRegex unresolved_regex = 8;
+    SortOrder sort_order = 9;
```

Comment:
Do you mind adding a bit of documentation that would explain what are valid uses of `SortOrder` in expressions? This is just to make sure it's understood that you can't use this in a projection etc.

If the only use-case for SortOrder is to be used in `Sort` relation and `Window` then maybe it's better to pull it out from the expressions message.

Reply:
I add a note for not support in projection, but I guess the developers should be aware of this.